dipy/stats/analysis.py

```diff
@@ -17,15 +17,15 @@
                                       orient_by_streamline,
                                       transform_streamlines,
                                       Streamlines)
-
+from glob import glob
 pd, have_pd, _ = optional_package("pandas")
 _, have_tables, _ = optional_package("tables")
 
 if have_pd:
     import pandas as pd
 
 
-def _save_hdf5(fname, dt, col_name, col_size=5):
+def _save_hdf5(fname, dt, col_name, col_size=16):
```

Comment:
Does this function belong in `dipy.io` instead?

Reply:
This is an internal function that should only be used while creating buan_bundle_profiles. This is not a generic function for saving all kinds of HDF5 files.

Should I move to dipy.io or keep it here? someone might try to use it as a generic function for saving HDF5 files if we have it in dipy.io.

Reply:
moving it to `dipy.io.utils` and renaming it to `save_buan_profiles_hdf5`